src/ext/gui.rs

```diff
@@ -26,4 +67,5 @@ pub struct clap_host_gui {
     pub resize: unsafe extern "C" fn(host: *const clap_host, width: u32, height: u32) -> bool,
     pub request_show: unsafe extern "C" fn(host: *const clap_host),
     pub request_hide: unsafe extern "C" fn(host: *const clap_host),
+    pub closed: unsafe extern "C" fn(host: *const clap_host, was_destroyed: bool),
```

Comment:
This struct is missing several changes which I believe I missed when updating to 0.18.0 (see https://github.com/free-audio/clap/blob/75dabe9c6c53192dc6249134c8bb9e562fa915a0/include/clap/ext/gui.h#L83-L98). Would you mind fixing them?

Reply:
Oops yeah I only went based on the diffs, I added the changes from https://github.com/free-audio/clap/commit/8fddd5b70352d01f9d5a1f23d4627a008d2feac4